requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy
 numba
 sqlalchemy
 rich
-pytables
+tables
```

Comment:
We may need to add a second version of the requirements file since we will want to support conda distribution of precovery as well. 

I think in conda its pytables and on pip its tables.

Reply:
I will update the github actions to build both for pypi and conda, as well as documentation.

Reply:
I'll be pushing for the pypi package to simply use `setup.cfg` and for the conda install to use what's specified in `meta.yml` if I'm understanding how it works, correctly. I don't believe we need the requirements files at all and this way we have distinct sets of requirements.